chain/indexer/CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Changelog
 
-# 0.2.0
+## 0.3.0
+
+* Add `ExecutionOutcomesWithReceipts` type
+* Add `IndexerExecutionOutcomeWithReceipt` struct that contains `ExecutionOutomeWithIdView` and corresponding `ReceiptView`
+* `IndexerTransactionWithOutcome` has `outcome` type of `IndexerExecutionOutcomeWithReceipt` now.
```

Comment:
It is too specific and does not help the end-users to understand what is the implication result for them, so I suggest the following:

```suggestion

### Breaking changes

* To extended the `receipt_execution_outcomes` with information about the corresponding receipt we had to break the API (the old outcome structure is just one layer deeper now [under `execution_outcome` field`])
 `ExecutionOutcomesWithReceipts` type
```